src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/TextEditor.cs

```diff
@@ -1562,7 +1562,7 @@ private object InitTextStore(object o)
                     }
 
                     _textstore.OnAttach();
-                    Marshal.ReleaseComObject(threadManager);
+                    ((IDisposable)threadManager).Dispose();
```

Comment:
Is this safe? Would this be better?
```suggestion
                    if (threadManager is IDisposable disposable)
                        disposable.Dispose();
```

Reply:
If `threadManager` is not our warpper, then we have something seriously wrong. And across codebase only `TF_CreateThreadMgr` produce `ITfThreadMgr`.